### ISSUE TITLE
Twig 1.x fails to load class 'Twig_Environment' on Windows

### ISSUE THREAD

**Issue by toxpal** [twigphp/Twig#3139]
This is a very similar issue as https://github.com/twigphp/Twig/issues/3092

The main difference in my situation is that Twig (always the latest version of 1.x branch) is manually bundled with script. That is to say, Composer isn't used, I download Twig libraries and bundle them with script. Works great. Except for some users who try to install script on localhost running Windows. They have the same error:

> Fatal error: Uncaught Error: Class 'Twig_Environment' not found

And this the code I use:

```
require_once(__DIR__."/lib/Twig/Autoloader.php");
Twig_Autoloader::register();
$loader=new Twig_Loader_Filesystem(__DIR__."/templates");
$twig=new Twig_Environment($loader);
```

So only the last line fails. I know it's not a problem with my code because it works great on Linux servers and also works great on 90% of Windows servers. However, some clients report exactly the same error (Class 'Twig_Environment' not found) when trying to use script on Windows servers. Some of them use XAMPP, some other use Laragon, so the issue is probably not related to specific software.

Personally, I couldn't reproduce the issue on my own Windows 7 + XAMPP machine, but I get reports about this error from customers (running Windows) from time to time. Which means the issue comes from Twig, and my best guess it's the autoloader.

**Comment by Tobion** [comment]
Considering the autoloader is already deprecated in 1.x, there is probably not much incentive for people to try to fix such an edge case situation that cannot be reproduced (not even by yourself).

**Comment by Tobion** [comment]
I'd suggest to try to use the composer autoloader. You can use it even if you don't use composer to install dependencies.

**Comment by toxpal** [comment]
Thanks for the info. However, if it's deprecated, shouldn't documentation be updated? Now it says to use autoloader - https://twig.symfony.com/doc/1.x/api.html

**Comment by stof** [comment]
The doc indeed deserves an update.

**Comment by fabpot** [comment]
Fixed in #3196
